scripts/script.js

```diff
@@ -0,0 +1,116 @@
+const newTitle = document.getElementById('new-title'); // eslint-disable-line max-classes-per-file
+const newAuthor = document.getElementById('new-author');
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class Library {
+  constructor() {
+    this.books = [
+      {
+        title: 'title1',
+        author: 'autor1',
+      },
+      {
+        title: 'title2',
+        author: 'autor2',
+      },
+      {
+        title: 'Dune',
+        author: 'Frank Herbert',
+      },
+    ];
+  }
+
+  addBook(bookTitle, bookAuthor) {
+    if (bookTitle !== '' && bookAuthor !== '') {
+      const newBook = new Book(bookTitle, bookAuthor);
+      this.books.push(newBook);
+
+      newTitle.value = '';
+      newAuthor.value = '';
+      loadBooks(); // eslint-disable-line no-use-before-define
```

Comment:
- [ ] Please take note you shouldn't disable linters using inline disable code like the above the only check you can disable for this specific project is the "multiple classes per file check" so please remove this disable comment. This applies to other linter disable comments as well. ✔